docs/installation/linux/gentoolinux.md

```diff
@@ -50,7 +50,7 @@ IRC channel on the Freenode network.
 | btrfs         |         |Enables dependencies for the "btrfs" graph driver, including necessary kernel flags.|
 | contrib       |  Yes    |Install additional contributed scripts and components.|
 | device-mapper |  Yes    |Enables dependencies for the "devicemapper" graph driver, including necessary kernel flags.|
-| doc           |         |Add extra documentation (API, Javadoc, etc). It is recommended to enable per package instead of globally.|
+| doc           |         |Add extra documentation (API, Javadoc, etc.) It is recommended to enable per package instead of globally.|
```

Comment:
I suggest rewording this to something like "Add extra documentation, such as API and Javadoc."